README.md

```diff
@@ -2,9 +2,9 @@
 
 An engineer working for a [CSE](./CSE.md) project...
 
-* Has responsibilities to their team – mentor, coach, and lead.
+* Has responsibilities to their team – mentor, coach, and dev lead.
 * Knows their **playbook**. Follows their playbook. Fixes their playbook if it is broken. If they find a better playbook, they copy it. If somebody could use your playbook, give them yours.
-* Leads by example. Models the behaviors we desire both interpersonally and technically.
+* Dev leads by example. Models the behaviors we desire both interpersonally and technically.
```

Comment:
Can you make all the mentions of "Dev Lead" and "Process Lead" capitalized so that it's more consistent as it's a bit inconsistent across different readmes.